src/platform/site-wide/sass/merger.scss

```diff
@@ -10,6 +10,10 @@
 
   .va-dropdown-panel {
     min-width: 102px;
+
+    p {
```

Comment:
This is pretty broad, and affects all our dropdown panels. There aren't a ton of those, but could we put this in a class specific to the help menu?

Reply:
I did earlier and was thinking the same, but @ncksllvn suggested using the vets class, but it didn't really work like I expected so I just added it here. I'll add this to this specific footer element.